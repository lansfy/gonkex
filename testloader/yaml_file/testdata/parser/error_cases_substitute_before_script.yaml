- name: loader MUST fail if cannot substitute beforeScriptArgs to request body
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  beforeScript:
    path: '/some/path/{{ .foo }'
  cases:
    - beforeScriptArgs:
        foo: "script1.sh"
