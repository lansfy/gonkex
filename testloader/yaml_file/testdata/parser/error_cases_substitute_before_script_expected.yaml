- Error: "process 'testdata/parser/error_cases_substitute_before_script.yaml': test 'loader MUST fail if cannot substitute beforeScriptArgs to request body #1': template: $.beforeScript.path:1: unexpected \"}\" in operand"
